competitor_url,error
https://Key Services:,"HTTPSConnectionPool(host='key%20services', port=443): Max retries exceeded with url: / (Caused by NameResolutionError(""<urllib3.connection.HTTPSConnection object at 0x101273e00>: Failed to resolve 'key%20services' ([Errno 8] nodename nor servname provided, or not known)""))"
